packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -73,12 +81,16 @@ contract DeployOwnership is Deploy {
         SafeConfig memory safeConfig = SafeConfig({ threshold: 10, owners: exampleCouncilOwners });
         councilConfig_ = SecurityCouncilConfig({
             safeConfig: safeConfig,
-            livenessInterval: 24 weeks,
-            thresholdPercentage: 75,
-            minOwners: 8,
-            fallbackOwner: mustGetAddress("FoundationSafe"),
-            deputyGuardian: makeAddr("DeputyGuardian"),
-            superchainConfig: SuperchainConfig(mustGetAddress("SuperchainConfig"))
+            livenessModuleConfig: LivenessModuleConfig({
+                livenessInterval: 24 weeks,
+                thresholdPercentage: 75,
+                minOwners: 8,
+                fallbackOwner: mustGetAddress("FoundationSafe")
+            }),
+            deputyGuardianModuleConfig: DeputyGuardianModuleConfig({
+                deputyGuardian: makeAddr("DeputyGuardian"),
```

Comment:
slightly safer default, and we know it's available since we use this as fallback owner also
?```suggestion
                deputyGuardian: mustGetAddress("FoundationSafe"),
```

Reply:
made this change locally.